core/src/test/resources/com/exadel/aem/toolkit/bundle/lists/util/customList.json

```diff
@@ -3,23 +3,23 @@
   "jcr:content": {
     "jcr:primaryType": "cq:PageContent",
     "jcr:title": "Custom List Page",
```

Comment:
Not "custom" anymore) And please recheck all other possible places where enitites are still named "custom"

Reply:
This one is a Custom list in a sense that it does not use SimpeItem's dialog and data structure, but a custom one :)